app/Platform/Services/GameListService.php

```diff
@@ -372,29 +379,24 @@ private function initializeUserGames(User $user, array $gameIds): void
 
     private function getTitleColumn(array $filterOptions): array
     {
+        $consoleName = null;
+        if ($this->withConsoleNames) {
+            $consoleName = $filterOptions['console'] ?? null;
+        }
+
         return [
             'header' => 'Title',
-            'render' => function ($game) use ($filterOptions) {
-                if (!$filterOptions['console']) {
-                    echo '<td class="py-2">';
```

Comment:
You requested I add this "py-2" because the multi-line rows felt "snug": https://github.com/RetroAchievements/RAWeb/pull/2130#discussion_r1442207108

If we don't need that, we could move the <td> elements out of render and put them directly in the sets-page.blade.php.

Reply:
I think I screwed up here; I'm seeing a regression.

Reply:
![Screenshot 2024-01-23 at 5 56 10 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/80f9f7f0-3c83-4417-aa0d-e13d944844d4)

Digging into this now.

Reply:
Sorry about that - what a mess.

There was both a style and a logic regression here. I think they're both cleaned up in latest.

`py-2` is now reapplied if `$consoleName` is truthy.

The `$consoleName` assignment logic has been adjusted to:

```php
$consoleName = null;
if ($this->withConsoleNames && empty($filterOptions['console'])) {
    $consoleName = $game['ConsoleName'] ?? null;
}
```

Now I correctly see console names again on hubs if I'm not grouping by console.